dtbase/tests/test_frontend_locations.py

```diff
@@ -0,0 +1,27 @@
+"""
+Test that the DTBase locations pages load
+"""
+from flask import url_for, request
+import pytest
+import requests_mock
+
+
+def test_new_location_schema_no_backend(frontend_client):
+    with frontend_client:
+        response = frontend_client.get(
+            "/locations/new_location_schema", follow_redirects=True
+        )
+        assert response.status_code == 200
+        html_content = response.data.decode("utf-8")
+        assert "Backend API not found" in html_content
+
+
+def test_new_location_schema(frontend_client):
+    with frontend_client:
+        with requests_mock.Mocker() as m:
+            m.get("http://localhost:5000/location/list_location_schemas", json=[])
+            m.get("http://localhost:5000/location/list_location_identifiers", json=[])
```

Comment:
Just for me to learn: Why are you doing these get requests?

Reply:
These are "mock" get requests.   With this, if anything in the code ends up sending a "get" request to those urls, it won't actually send the request, but instead the mocker will return whatever is in the "json" argument (in this case, empty lists).    
This is a way of testing the frontend, without needing to have the backend running...